### ISSUE TITLE
What's the process for previous attendees to update their older profile? 

### ISSUE THREAD

**Issue by jeddc** [nickyhajal/world-domination-summit#96]
Do I just send them their unique link? Is it the same as their previous year?

**Comment by nickyhajal** [comment]
What do you mean update their profile? As in, they went to 2015, they have a ticket for 2016 and they want to update their details?

**Comment by jeddc** [comment]
Yup. That's exactly it. Or if an attendee had a profile last year under one email, but then bought a ticket this year with a new email. How would I best support them?

**Comment by jeddc** [comment]
Here's some feedback from a current attendee who tried to update their profile. I gave them their unique link to go through the process: 

_That link worked pretty similar to the other one. It sent me to a page where the only option is to create a new profile. What was different was that when I put in my username, it told me the user already existed, flashed a red error, then sent me to the next screen. The "Choose your interests" overlay obscured most of the rest of the windows/questions, but I was able to click my way through it. However, the selections I made for "Choose your interests" did not get saved. They are still the ones from last year. And the label over the text box says "Why are you excited about coming to WDS 2015" although it did save my answer to the question.

It did save my Instagram, Twitter and Facebook updates (although I am pretty sure Twitter and Facebook were already set previously). It appears that something overwrote my email address (which I think used to be leila@sevana.us) with duo@sevana.us. I can't log in using either leila@sevana.us or duo@sevana.us. I can only log in using my username. There doesn't seem to be any way for me to edit my email address. 

I think that it might be more straightforward to send returning people to the login page at https://worlddominationsummit.com/login, although things still seem pretty wonky to me. It keeps wanting to have me set a password, and there is no way to get past that page without doing so. I think I've set my password six or seven times this morning already, so it should be pretty safe by now :-)

I had been using Chrome on linux, so I just tried it using Firefox on windows to see if things worked any better. They seem to work about the same. I can't find any way to get back to choosing my interests. So things are a little better, but I think they still need some work. I'm sending the screenshot of how the one overlay obscured the text and the buttons. The only way I could get past that was to click on the little faint window preview on the right.

![wdssignup](https://cloud.githubusercontent.com/assets/11277509/15296278/8bf3d34c-1b49-11e6-9b12-ca51fc28b745.png)

_

Any thoughts? Thanks Nicky!
